CatFoodCalc/cat_calc.py

```diff
@@ -1,43 +1,36 @@
-class Cat:
-    # age = {
-    #     1: "kitten under 4 mo",
-    #     2: "kitten under 9 mo",
-    #     3: "kitten under 12 mo",
-    #     4: "adult cat",
-    #     5: "pregnant cat",
-    #     6: "lactating cat (under 3 kittens)",
-    #     7: "lactating cat (3+ kittens)",
-    #     8: "senior cat"
-    # }
-    # activity = {
-    #     1: "neutered, inactive and indoor cats",
-    #     2: "active and outdoor cats"
-    # }
-
-    kcal_per_kg_indoor = [52, 75]
-    kcal_per_kg_outdoor = [100]
+import presets
+
+class Cat(object):
```

Comment:
It is a nice habit to give your class something to inherit from, even if it's just most basic Python object